lib/display.go

```diff
@@ -29,23 +29,51 @@ import (
 	"github.com/fatih/color"
 )
 
-var layout = `{{if .Alias}}{{.Alias}}
+var layout = `
+{{- define "PkixName" -}}
+{{- if .CommonName}}
+	CommonName: {{.CommonName}}
+{{- end -}}
+{{- range .Organization}}
+	Organization: {{.}}
+{{- end -}}
+{{- range .OrganizationalUnit}}
+	OrganizationalUnit: {{.}}
+{{- end -}}
+{{- range .Country}}
+	Country: {{.}}
+{{- end -}}
+{{- range .Locality}}
+	Locality: {{.}}
+{{- end -}}
+{{- range .Province}}
+	Province: {{.}}
+{{- end -}}
+{{- range .StreetAddress}}
+	StreetAddress: {{.}}
+{{- end -}}
+{{- range .PostalCode}}
+	PostalCode: {{.}}
+{{- end -}}
+{{- range .ExtraNames -}}
+	{{- range $index, $type := .Type }}
+		{{- if $index }}.{{else}}
```

Comment:
This is printing the OID.  So on the first entry, we print \n\t, and then seperate with dots

So we end up with something that looks like
```
        0.1.2.3: Foo
        5.6.7345.9: Bar
```

I should include a test case for this -- I'll add one